operate/client/src/App/Processes/ListView/InstancesTable/index.tsx

```diff
@@ -47,6 +47,13 @@ const InstancesTable: React.FC = observer(() => {
     state: {status, filteredProcessInstancesCount, processInstances},
   } = processInstancesStore;
 
+  /**
+   * Is true if at least one process instances from the store has a version tag.
+   */
+  const hasVersionTags = processInstances.some(({processId}) => {
+    return processesStore.getVersionTag(processId);
+  });
```

Comment:
Now I'm checking here if there is at least one process instance in the store with a related version. If there is none found, the version tag column header won't be shown.